workspace_visualization/src/capability_map_visual.cpp

```diff
@@ -0,0 +1,156 @@
+#include <OGRE/OgreVector3.h>
+#include <OGRE/OgreSceneNode.h>
+#include <OGRE/OgreSceneManager.h>
+
+#include <rviz/ogre_helpers/shape.h>
+#include <rviz/display_context.h>
+#include <rviz/display_factory.h>
+
+#include<tf2/LinearMath/Quaternion.h>
+
+#include "capability_map_visual.h"
```

Comment:
All of the header files for the workspace_visualization package need to be put into the include/workspace_visualization folder to follow normal ROS package conventions.